lib/l10n/app_en.arb

```diff
@@ -1,5 +1,6 @@
 {
   "appTitle": "Tazkrtak",
   "common_loading": "Loading...",
-  "error_generic": "Something went wrong!"
+  "error_generic": "Something went wrong!",
+  "register": "Register"
```

Comment:
Use this naming convention: `<featureName>_<resourceName>`, becuase this file will get messy later
e.g.: `register_title`